src/vs/workbench/contrib/chat/browser/chat.contribution.ts

```diff
@@ -200,26 +200,43 @@ configurationRegistry.registerConfiguration({
 			tags: ['experimental', 'onExp']
 		},
 		[PromptsConfig.CONFIG_KEY]: {
-			type: 'object',
+			type: 'boolean',
 			title: nls.localize(
-				'chat.promptFiles.config.title',
+				'chat.reusablePrompts.config.enabled.title',
 				"Prompt Files",
 			),
 			markdownDescription: nls.localize(
-				'chat.promptFiles.config.description',
+				'chat.reusablePrompts.config.enabled.description',
+				"Enable reusable prompt files (`*{0}`) in Chat, Edits, and Inline Chat sessions. [Learn More]({0}).",
+				PROMPT_FILE_EXTENSION,
+				DOCUMENTATION_URL,
+			),
+			default: true,
+			restricted: true,
+			disallowConfigurationDefault: true,
+			tags: ['experimental', 'prompts', 'reusable prompts', 'prompt snippets', 'instructions'],
```

Comment:
We have a small set of tags, these show up with `@tags` in the settings editor. Not sure whether you mean for all of these to be added

Reply:
Yeah I expected (and I've checked that) it uses only the `experimental` tag from the list in the UI. I wanted to add the other tags as a documentation of all other names we were calling the feature, and I hoped that maybe it will aid with search through the settings (it seems to not 😞).

Reply:
Ah I see